packages/babel-parser/src/util/scopeflags.ts

```diff
@@ -1,102 +1,90 @@
 // Each scope gets a bitset that may contain these flags
-// prettier-ignore
-export const SCOPE_OTHER        = 0b000000000,
-             SCOPE_PROGRAM      = 0b000000001,
-             SCOPE_FUNCTION     = 0b000000010,
-             SCOPE_ARROW        = 0b000000100,
-             SCOPE_SIMPLE_CATCH = 0b000001000,
-             SCOPE_SUPER        = 0b000010000,
-             SCOPE_DIRECT_SUPER = 0b000100000,
-             SCOPE_CLASS        = 0b001000000,
-             SCOPE_STATIC_BLOCK = 0b010000000,
-             SCOPE_TS_MODULE    = 0b100000000,
-             SCOPE_VAR = SCOPE_PROGRAM | SCOPE_FUNCTION | SCOPE_STATIC_BLOCK | SCOPE_TS_MODULE;
+/* eslint-disable prettier/prettier */
+export const enum ScopeFlag {
+  OTHER = 0b000000000,
```

Comment:
Nit: can you vertically align them? It makes it easier to see the position of `1`.